src/main/resources/db-scripts/cgds.sql

```diff
@@ -755,5 +756,5 @@ CREATE TABLE `resource_study` (
 );
 
 -- THIS MUST BE KEPT IN SYNC WITH db.version PROPERTY IN pom.xml
-INSERT INTO info VALUES ('2.13.1', NULL);
+INSERT INTO info VALUES ('2.13.1','hgnc_v2023.10.1', NULL );
```

Comment:
This version should maybe be `N/A` or empty string. Whoever loads the gene table data should set the version